src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -78,6 +81,11 @@ object ConsignmentFields {
         "transferringBody",
         OptionType(TransferringBodyType),
         resolve = context => DeferConsignmentBody(context.value.consignmentid)
+      ),
+      Field(
+        "fileMetadata",
```

Comment:
Should this be called `files`, and each file has its own metadata? I'd have expected the GraphQL graph to be something like:

```
consignment {
  files {
    id
    metadata {
      legalStatus,
      sha256ClientSideChecksum
    }
  }
}
```

Reply:
Very good point. If we have a files object we can add other none metadata related stuff to the object as well. I've changed it.